### PR TITLE
Policy

### PR DESCRIPTION
**Pull request by avestuk**

### CODE REVIEW

File: _docs/operations/policies.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+layout: guide
+title: StorageOS Docs - Policies
+anchor: operations
+module: operations/policies
+---
+
+# Policies
+
+Policies are a way to grant access to StorageOS namespaces. Policies can be
```
Comment (Arau) [inline _docs/operations/policies.md]: Policies are a way to grant  --> Policies control access to

File: _docs/operations/policies.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+configured at the group or user level so access can be controlled granularly.
+
```
Comment (Arau) [inline _docs/operations/policies.md]: Nowhere says why a policy is used or helpful. Do we have a concept page for this?

I'd add a small note in the 1 paragraph mentioning that 

"the Administrator can restrict what namespaces each user can see to ensure privacy and isolation of responsibilities"

**Comment by actionbuddha** [comment]
Note that we now have a space in the concepts section for policies (see https://github.com/storageos/storageos.github.io/pull/228) - so we can refactor the info here as appropriate. I suggest we keep this PR just for the operational piece to faciliate easy merging.

File: _docs/operations/policies.md (hunk 2 above)
Comment (avestuk) [reply _docs/operations/policies.md]: I think that's more of a job for the concepts policy page. I made to to clearly spell out that users only have access to namespaces that they are mapped to with policies, except for the default namespace.

**Pull request merged**